test/e2e/util/util.go

```diff
@@ -85,6 +85,10 @@ var expectedKasManagementComponents = []string{
 	"cloud-controller-manager",
 	"olm-collect-profiles",
 	"aws-ebs-csi-driver-operator",
+	"azure-disk-csi-driver-operator",
```

Comment:
Got the list by this search: https://github.com/search?q=repo%3Aopenshift%2Fcluster-storage-operator%20hypershift.openshift.io%2Fneed-management-kas-access&type=code